README.md

```diff
@@ -54,7 +57,9 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:residents].pop[1]
+
```

Comment:
use `delete_at` instead of pop. Also `[]` are for arrays and for Hash-keys in ruby. I think you meant to use `()`